### PR TITLE
Add Home Assistant Green on Amazon link to buy box

### PR DESCRIPTION
**Pull request by Nikola-Kovic**
## Proposed change

I added Nabu Casa Amazon shop to our list of distributors 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: -
- Link to parent pull request in the Brands repository: -
- This PR fixes or closes issue: fixes -

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Updated shipping options on the product webpage to include links for both Amazon and ameriDroid, providing users with clearer choices for purchasing.
- **Content Updates**
  - Changed the display text for shipping options to enhance clarity, ensuring users understand where to direct their purchases.

### CODE REVIEW

**Comment by b-uwe** [comment]
@Nikola-Kovic I just realized we started off naming this "Amazon ..." and we placed it before "AmeriDroid". Now it's named "Nabu Casa via Amazon" which means, it's not alphabetically anymore 🤦.

I believe we should stick to alphabetical order!

File: source/green/index.html [hunk 1]
```diff
@@ -2312,11 +2312,11 @@ <h1 class="heading-large">
       <details id="dist_america">
         <summary class="region">Shipping to America</summary>
         <div>
-          <a href="https://ameridroid.com/products/home-assistant-green" target="_blank">
+          <a href="https://a.co/d/j7bjZq3" target="_blank">
             <div class="distributor">
               <div>
                 <div>🇺🇸</div>
-                <div>ameriDroid</div>
+                <div>Nabu Casa via Amazon</div>
```
Comment (c0ffeeca7) [inline source/green/index.html]:
```suggestion
                <div>Amazon US</div>
```

**Review by c0ffeeca7 — DISMISSED** [review]

File: source/green/index.html [hunk 2]
```diff
@@ -2326,10 +2326,25 @@ <h1 class="heading-large">
...
+                    d="M8.59,16.58L13.17,12L8.59,7.41L10,6L16,12L10,18L8.59,16.58Z"
+                  />
+                </svg>
+              </div>
+            </a>
+  
           <a href="https://cloudfree.shop/product/home-assistant-green/" target="_blank">
             <div class="distributor">
-              <div>
+              <div>
```
Comment (balloob) [inline source/green/index.html]:
```suggestion
              <div>
```

File: source/green/index.html [hunk 3]
```diff
@@ -2312,11 +2312,11 @@ <h1 class="heading-large">
       <details id="dist_america">
         <summary class="region">Shipping to America</summary>
         <div>
-          <a href="https://ameridroid.com/products/home-assistant-green" target="_blank">
+          <a href="https://a.co/d/j7bjZq3" target="_blank">
```
Comment (balloob) [inline source/green/index.html]:
```suggestion
          <a href="https://www.amazon.com/dp/B0CXVKSG19?maas=maas_adg_BB51722E249D328CABB2F6DF1F8976CC_afap_abs&ref_=aa_maas&tag=maas" target="_blank">
```

**Review by balloob — APPROVED** [review]

**Review by balloob — APPROVED** [review]

**Pull request merged**